pallets/pooled-staking/src/lib.rs

```diff
@@ -754,4 +757,30 @@ pub mod pallet {
             pools::distribute_rewards::<T>(&candidate, rewards)
         }
     }
+    impl<T: Config> tp_traits::NotifyCollatorOnlineStatusChange<Candidate<T>> for Pallet<T> {
+        fn is_collator_in_sorted_eligible_candidates(collator: &Candidate<T>) -> bool {
+            <SortedEligibleCandidates<T>>::get()
+                .into_iter()
+                .any(|c| c.candidate == collator.clone())
```

Comment:
Can it be `&c.candidate == collator` to avoid cloning?

Reply:
Yup updated it in https://github.com/moondance-labs/tanssi/pull/881/commits/8f345e6068d5edd50e2809f61c32e6b0d3732c7b